--- ./current_players_row.inc	2020-11-19 19:20:34.000000000 +0000
+++ ./current_players_row.inc	2021-06-30 21:34:29.008744773 +0100
@@ -1,7 +1,9 @@
 <tr class="<%evenodd%>">
   <td style="background: <%player.teamcolor%>; color: <%player.teamcolor%>;"><%player.teamid%>&#160;</td>
   <td><%player.name%></td>
-  <td class="right"><%player.ping%></td>
+  <td><%player.perk.name%></td>
+  <td class="center"><%player.perk.level%></td>
+  <td class="center"><%player.ping%></td>
   <td><%player.ip%></td>
   <td><%player.uniqueid%></td>
   <td><%player.steamid%></td>
