diff --git a/current_rules.inc b/current_rules.inc
old mode 100755
new mode 100644
index d3ef5db..da14fb2
--- a/current_rules.inc
+++ b/current_rules.inc
@@ -1,10 +1,12 @@
-<dt>Players</dt>
-<dd><%rules.numplayers%>/<%rules.maxplayers%></dd>
-<dt>Minimum to Start</dt>
-<dd><%rules.minnetplayers%></dd>
-<dt>Spectators</dt>
-<dd><%rules.numspectators%>/<%rules.maxspectators%></dd>
-<dt>Player Map Voting</dt>
-<dd><%rules.mapvote%></dd>
-<dt>Player Kick voting</dt>
-<dd><%rules.kickvote%></dd>
+<dt>Wave</dt>
+<dd><%wave.num%></dd>
+<dt>Players</dt>
+<dd><%rules.numplayers%>/<%rules.maxplayers%></dd>
+<dt>Minimum to start</dt>
+<dd><%rules.minnetplayers%></dd>
+<dt>Spectators</dt>
+<dd><%rules.numspectators%>/<%rules.maxspectators%></dd>
+<dt>Map vote</dt>
+<dd><%rules.mapvote%></dd>
+<dt>Kick player</dt>
+<dd><%rules.kickvote%></dd>
