diff --git a/current_player_row.inc b/current_player_row.inc
old mode 100755
new mode 100644
index 9ba6537..c137593
--- a/current_player_row.inc
+++ b/current_player_row.inc
@@ -1,10 +1,11 @@
-<tr class="<%evenodd%>">
-  <td style="background: <%player.teamcolor%>; color: <%player.teamcolor%>;"><%player.teamid%>&#160;</td>
-  <td><%player.name%></td>
-  <td><%player.perk.name%></td>
-  <td class="right"><%player.score%></td>
-  <td class="right"><%player.pawn.health%></td>
-  <td class="right"><%player.kills%></td>
-  <td class="right" title="Packet loss: <%player.packetloss%>"><%player.ping%></td>
-  <td class="center"><%player.admin%></td>
-</tr>
+<tr class="<%evenodd%>">
+  <td style="background: <%player.teamcolor%>; color: <%player.teamcolor%>;"><%player.teamid%>&#160;</td>
+  <td><%player.name%></td>
+  <td><%player.perk.name%></td>
+  <td class="right"><%player.perk.level%></td>
+  <td class="right"><%player.score%></td>
+  <td class="right"><%player.pawn.health%></td>
+  <td class="right"><%player.kills%></td>
+  <td class="right" title="Packet loss: <%player.packetloss%>"><%player.ping%></td>
+  <td class="center"><%player.admin%></td>
+</tr>
