diff --git a/gamesummary.inc b/gamesummary.inc
index cd4e1f7..4b47e1f 100755
--- a/gamesummary.inc
+++ b/gamesummary.inc
@@ -4,6 +4,6 @@
   <dd class="gs_map"><%map.title.safe%></dd>
   <dt class="gs_players">Players</dt>
   <dd class="gs_players"><%rules.numplayers%>/<%rules.maxplayers%></dd>
-  <dt class="gs_time">Time</dt>
-  <dd class="gs_time"><%time.elapsed%>/<%time.timelimit%></dd>
+  <dt class="gs_wave">Wave <%wave.num%>/<%wave.max%></dt>
+  <dd class="gs_wave"><%wave.monsters.dead%>/<%wave.monsters.total%></dd>
 </dl>
\ No newline at end of file
