diff --git a/header_base.inc b/header_base.inc
old mode 100755
new mode 100644
index 217a460..d508200
--- a/header_base.inc
+++ b/header_base.inc
@@ -1,18 +1,19 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
-<head>
-	<title>Killing Floor 2 WebAdmin - <%page.title%></title>
-	<link rel="stylesheet" type="text/css" href="/images/kf2.css?gzip" />
-	<link rel="stylesheet" type="text/css" href="/images/kf2modern.css?gzip" />
-	<%webadminskins.meta%>
-	<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
-	<meta name="generator" content="Killing Floor 2 WebAdmin" />
-	<script type="text/javascript">
-	var webadminPath = '<%webadmin.path%>';
-	var pageUri = '<%page.fulluri%>'; 
-	</script>
-	<script type="text/javascript" src="/images/jquery.js?gzip"></script>
-	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,400italic,600,700,700italic,600italic' rel='stylesheet' type='text/css'>
-	<%html.headers%>
-</head>
-<body class="<%page.css.class%>">
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
+<head>
+	<!-- KF2-MA-INSTALLED-FLAG -->
+	<title>Killing Floor 2 WebAdmin - <%page.title%></title>
+	<link rel="stylesheet" type="text/css" href="/images/kf2.css?gzip" />
+	<link rel="stylesheet" type="text/css" href="/images/kf2modern.css?gzip" />
+	<%webadminskins.meta%>
+	<link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
+	<meta name="generator" content="Killing Floor 2 WebAdmin" />
+	<script type="text/javascript">
+	var webadminPath = '<%webadmin.path%>';
+	var pageUri = '<%page.fulluri%>'; 
+	</script>
+	<script type="text/javascript" src="/images/jquery.js?gzip"></script>
+	<link href='http://fonts.googleapis.com/css?family=Open+Sans:400,400italic,600,700,700italic,600italic' rel='stylesheet' type='text/css'>
+	<%html.headers%>
+</head>
+<body class="<%page.css.class%>">
